### PR TITLE
Add Date to Link Liquid Tag

### PR DESCRIPTION
**Pull request by 12vanblart**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
I noticed an inconsistency where the publish date was not included in liquid links to dev posts.

**Disclaimer:** I'm not familiar with ruby at all and was struggling to get it working to run the tests (Windows 10 - locked out of installing WSL). I was also unable to get the liquid tags working in Gitpod [(Twitter thread with possibly helpful info)](https://twitter.com/jankeromnes/status/1138410459818790912). 

## Related Tickets & Documents
n/a

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
n/a (See above comments about Gitpod)

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

**Comment by mariocsee** [comment]
Unfortunately I'm unfamiliar with developing on Windows and don't use Gitpod. In my case, I would run `bin/rspec spec/liquid_tags/link_tag_spec.rb` on command line in MacOS.

The Travis build shows the results from running `link_tag_spec.rb` [here](https://travis-ci.com/thepracticaldev/dev.to/builds/115115921#L1026). 

It looks like the publish date needs some formatting 
> ex. `<h3>Chase Danger ・ 2019-06-12 00:22:36 +0900 ・ 1 min read</h3>` 

and the `correct_link_html(article)` method in `link_tag_spec.rb` needs to be updated.

File: app/views/articles/_liquid.html.erb [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <a href='<%= article.path %>' class='ltag__link__link'>
     <div class='ltag__link__content'>
       <h2><%= title %></h2>
-      <h3><%= article.user.name %> ・ <%= article.reading_time %> min read</h3>
+      <h3><%= article.user.name %> ・ <%= article.published_at %> ・ <%= article.reading_time %> min read</h3>
```
Comment (rhymes) [inline app/views/articles/_liquid.html.erb]: Hi @12vanblart, I think you should use `article.readable_publish_date` for consistency

Comment (12vanblart) [reply app/views/articles/_liquid.html.erb]: That sounds like a more appropriate option - Updated!

**Comment by 12vanblart** [comment]
Travis is showing a failure [here](https://travis-ci.com/thepracticaldev/dev.to/builds/115133812#L1026) - but I'm not sure what is causing it. 

Based on some quick google searches, it seems to be unable to find the edit button, but I don't believe my changes would have impacted that?

**Comment by maestromac** [comment]
@12vanblart Don't worry about that. It's a flaky spec.

**Comment by mariocsee** [comment]
LGTM! 👍

**Pull request merged**